### PR TITLE
BugFixes for python3

### PR DESCRIPTION
**Pull request by IvarsKarpics**

### CODE REVIEW

File: gui/utils/QtImport.py [hunk 1]
```diff
@@ -138,6 +131,15 @@
     except BaseException:
         pass
 
+    try:
+        from PyQt5.QtWebKit import *
```
Comment (rhfogh) [inline gui/utils/QtImport.py:135]: Should this be 'except ImportError:'?

**Review by rhfogh — CHANGES_REQUESTED** [review]
Approved - subject to my comment being either addressed or dismissed.

File: gui/utils/QtImport.py (hunk 1 above)
Comment (IvarsKarpics) [reply gui/utils/QtImport.py:135]: I changed the exception to ImportError

**Review by rhfogh — APPROVED** [review]

**Comment by IvarsKarpics** [comment]
If it is approved, can you merge it?

**Pull request merged**